### DISCUSSION TITLE
How add a custom field in "auth" response?

### DISCUSSION THREAD

**Discussion by clauxs88-dev** [delphi-blocks/WiRL#53] [General]
How do I add a field in the output attached to the JWT, example "expire_time"?
![image](https://github.com/delphi-blocks/WiRL/assets/5374560/9d624f5a-5d0f-4f54-b3f1-e2a5d098fa86)
![image](https://github.com/delphi-blocks/WiRL/assets/5374560/23a8b31a-23fe-4a60-8683-f67915bb3b46)

**Comment by fperana** [comment]
If what you're trying to achieve is set an expiry time for the JWT, this info must be added to the JWT itself by setting a custom "claim". This ensures the custom data couldn't be modified by the user, as this would invalidate the JWT signature.
